File: AngularLovelyjubbly/ClientApp/src/app/shared/models/qbrating.ts
import { ITournament } from './tournament';
import { ITeam } from './team';

export interface IQBRating {
    qbRatingId: number;
    tournament: ITournament;
    tournamentShort: number;
    team: ITeam;
    completion: number;
    gain: number;
    touchdown: number;
    interception: number;
    rating: number;
}

export class QBRating implements IQBRating {

    constructor(public qbRatingId: number = 0,
        public tournament: ITournament = null,
        public tournamentShort: number = null,
        public team: ITeam = null,
        public completion: number = null,
        public gain: number = null,
        public touchdown: number = null,
        public interception: number = null,
        public rating: number = null) {
    }
}

File: AngularLovelyjubbly.Data.Sql.Models/QBRating.cs
namespace AngularLovelyjubbly.Data.Sql.Models
{
    public partial class QBRating
    {
        public QBRating()
        {
        }

        public int QBRatingId { get; set; }
        //foreign key
        public int TournamentId { get; set; }
        //navigation property
        public virtual Tournament Tournament { get; set; }
        //foreign key
        public int TeamId { get; set; }
        //navigation property
        public virtual Team Team { get; set; }
        public double Completion { get; set; }
        public double Gain { get; set; }
        public double Touchdown { get; set; }
        public double Interception { get; set; }
    }
}
